tests/queries/0_stateless/03727_tolowcardinality_nullable_cast.sql

```diff
@@ -0,0 +1,48 @@
+-- Tags: no-fasttest
```

Comment:
Why no fast test? Fast tests are quite useful in knowing quickly if something breaks in the future.